db/schema.rb

```diff
@@ -10,14 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170509072215) do
+ActiveRecord::Schema.define(version: 20170509160431) do
 
   create_table "active_funkis_shift_limits", force: :cascade do |t|
     t.integer  "active_limit", default: 0
     t.datetime "created_at",               null: false
     t.datetime "updated_at",               null: false
   end
 
+  create_table "artist_lineups", force: :cascade do |t|
```

Comment:
What is this empty table? (no need to remove by migrations, just wondering why its here)